client/hbs/files.hbs

```diff
@@ -0,0 +1,31 @@
+<html class="admin">
+{{> head.hbs}}
+
+{{#if user}}
+	<header>
+		<h1>File Browser</h1>
+		<a id="filePageTrigger">
+			<dttw-icon id="upload" class="material-icons">file_upload</dttw-icon>
```

Comment:
Actually, looks like I'm wrong, but I don't think that there's a case where `<dttw-icon>` shouldn't be `.material-icons`, so I'm not sure why it's set up this way.  We'll handle this later, I guess?